File: raw/github/paypal/paypal-ios-sdk-demo-app/snapshots/2026-08-16-047a50e/files/PayPalDemo/CardCheckoutViews/OrderCompleteView.swift
import SwiftUI

struct OrderCompleteView: View {
    let orderID: String
    var onDone: () -> Void
    
    var body: some View {
        VStack(alignment: .leading, spacing: 16) {
            SectionHeader(title: "Order Complete")
                .padding(.bottom, 25)
            Text("Thank you for your order! Your order number is \(orderID)")
                .font(.subheadline)
                .padding(.bottom)
            Spacer()
            SubmitButton(title: "Done", action: onDone)
        }
        .padding()
        .background(Color.white)
    }
}

File: raw/github/paypal/paypal-android-sdk-demo-app/snapshots/2026-08-16-d1137d5/files/app/src/main/java/com/firstapp/paypaldemo/main/OrderCompleteView.kt
package com.firstapp.paypaldemo.main

import androidx.compose.foundation.layout.Arrangement
import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.Spacer
import androidx.compose.foundation.layout.fillMaxWidth
import androidx.compose.foundation.layout.height
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.material3.Button
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp

@Composable
fun OrderCompleteView(
    orderId: String?,
    onDone: () -> Unit
) {

    Column(
        modifier = Modifier
            .padding(horizontal = 20.dp),
        verticalArrangement = Arrangement.Bottom,
    ) {
        Text("Order Complete", fontWeight = FontWeight.Bold, fontSize = 25.sp)
        Spacer(modifier = Modifier.height(24.dp))

        var successMessage = "Thank you for your order!"
        if (orderId != null) {
            successMessage += " Your order number is $orderId."
        }
        Text(successMessage)
        Spacer(modifier = Modifier.weight(1.0f))
        Button(
            onClick = onDone,
            modifier = Modifier
                .fillMaxWidth()
                .height(50.dp),
            shape = RoundedCornerShape(10.dp)
        ) {
            Text("Done")
        }
    }
}
